packages/drivers/routerlicious-driver/src/restWrapper.ts

```diff
@@ -107,8 +108,10 @@ export class RouterliciousRestWrapper extends RestWrapper {
 
         return {
             ...requestHeaders,
+            // TODO: replace header names with CorrelationIdHeaderName and DriverVersionHeaderName from services-client
             // NOTE: Can correlationId actually be number | true?
             "x-correlation-id": correlationId as string,
+            "x-driver-version": driverVersion,
```

Comment:
Curious: in `documentDeltaConnection.ts`, the `relayUserAgent` included both the `driverVersion` as well as the `client.details.environment`. Here, we seem to only use the `driverVersion` as a header. I was curious why we don't need the `client.details.environment` here.

Reply:
`client` in `client.details.environment` is not known to restwrapper, as it is a parameter to `connectToDeltaStream` and is therefore specific to the websocket codepath